src/components/fields/multi-select/types.ts

```diff
@@ -0,0 +1,9 @@
+import { InputMultiSelectProps } from "@lifesg/react-design-system";
+import { BaseFormElementProps } from "@lifesg/react-design-system/form/types";
+import { IFrontendEngineBaseFieldJsonSchema, TFrontendEngineBaseFieldJsonSchemaKeys } from "../../frontend-engine";
+
+export interface IMultiSelectRef extends BaseFormElementProps, InputMultiSelectProps<unknown, unknown> {}
+
+export interface IMultiSelectSchema
+	extends IFrontendEngineBaseFieldJsonSchema<"multiselect">,
+		Omit<InputMultiSelectProps<unknown, unknown>, TFrontendEngineBaseFieldJsonSchemaKeys> {}
```

Comment:
instead of `InputMultiSelectProps<unknown, unknown>` it should be `InputMultiSelectProps<IOption, IOption>`